webapp/update.py

```diff
@@ -44,6 +58,29 @@ def _generate_spec_rows_for_folders(drive: Drive, folders: List[Dict]):
         print(f"Found {len(files)} documents in {folder['name']}")
 
         for file_ in files:
+            if file_["id"] in existing_specs:
+                file_modification_date = datetime.datetime.fromisoformat(
+                    file_["modifiedTime"][:-1]
+                )
+                row = existing_specs.get(file_["id"])
+                previous_modified_date = row["values"][10]["formattedValue"]
+                previous_modified_date = datetime.datetime.fromisoformat(
+                    previous_modified_date[:-1]
+                )
+                if previous_modified_date >= file_modification_date:
+                    # file has not been modified since last update
+                    # use the same row as before
+                    try:
+                        new_row = [
+                            value.get("formattedValue", "")
+                            for value in len(row["values"])
```

Comment:
I think there's a small bug:
```suggestion
                            for value in row["values"]
```